src/Approvals/SFA.DAS.Approvals/Application/BulkUpload/Commands/BulkUploadUpdateLogResult.cs

```diff
@@ -0,0 +1,9 @@
+﻿using SFA.DAS.Approvals.InnerApi.Responses;
+
+namespace SFA.DAS.Approvals.Application.BulkUpload.Commands
+{
+    public class BulkUploadUpdateLogResult
+    {
+        public int LogId { get; set; }
+    }
+}
```

Comment:
sorry, see previous comments about unused code

Reply:
hi paul, removed this as part of removing all Update endpoint